node-graph/nodes/gstd/src/text.rs

```diff
@@ -25,7 +28,8 @@ fn text<'i: 'n>(
 	#[default(0.)]
 	tilt: f64,
 	align: TextAlign,
-	/// Splits each text glyph into its own row in the table of vector geometry.
+	/// When enabled, outputs a table with one Vector per character, allowing individual character manipulation.
+    /// When disabled, outputs a single Vector containing all characters combined. Default is disabled.
```

Comment:
"Vector" -> "vector element"

Remove sentence about default, that will be shown automatically as part of future documentation generation infrastructure.